docs/get_started/raspbian_setup.md

```diff
@@ -0,0 +1,72 @@
+# Installing MXNet on Ubuntu
+MXNet currently supports the Raspbian operating system for Raspberry Pi Devices, offering a python API. We currently reccomend running MxNet on the Raspberry Pi 3 or other devices with more than 1GB of RAM and with an SD card that has at least 4 GB free.
+The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MxNet library (though they can run the mxnet amalgamation library). 
+
```

Comment:
May be good idea to add a line about approx. memory usage by installing mxnet? This might give users an idea of what they are installing and what it takes on hardware